templates/admin.php

```diff
@@ -8,29 +8,40 @@
 <div class="section" id="richdocuments">
 	<h2>
 		<?php p($l->t('Collabora Online')) ?>
-		<a target="_blank" rel="noreferrer" class="icon-info"
-			title="<?php p($l->t('Open documentation'));?>"
-			href="https://nextcloud.github.io/richdocuments/app_settings"></a>
 	</h2>
-
 	<br/>
-	<label for="wopi_url"><?php p($l->t('Collabora Online server')) ?></label>
-	<input type="text" name="wopi_url" id="wopi_url" value="<?php p($_['wopi_url'])?>" style="width: 300px; display: block;">
-	<em><?php p($l->t('URL (and port) of the Collabora Online server that provides the editing functionality as a WOPI client.')) ?></em>
-	<br/><button type="button" id="wopi_apply"><?php p($l->t('Apply')) ?></button>
+	<em><?php p($l->t('Collabora Online is a powerful LibreOffice-based online office suite with collaborative editing, which supports all major documents, spreadsheet and presentation file formats and works together with all modern browsers.')) ?></em>
```

Comment:
Since you are on it, it would be nicer to wrap each paragraph in a `<p>` element and apply some proper margins for the spacing instead of line breaks.